zadanie3/Zadanie3.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Created by kamilbest on 08.06.15.
+ */
+
+import java.util.Scanner;
+import java.lang.Exception;
+
+public class Zadanie3 {
+
+    public static void main(String[] args) {
+        graWKosci gra = new graWKosci();
+        gra.start();
+    }
+}
+
+class graWKosci {
+    /**
+     * Sprawdzenie poprawnosci danych wejsciowych.
+     * Wyrzucenie wyjatku w przypadku blednych danych.
+     */
+
+    private void sprawdzPoprawnoscRzutu(int rzut) throws Exception {
+        if (rzut < 1 || rzut > 6)
+            throw new Exception("Bledne dane, mozesz uzyc liczb z przedzialu 1-6");
+    }
+
+    /**
+     * Obsluga tury aktualnego gracza
+     */
+    private int tura(Gracz g) {
+        int numer_rzutu = 1;
+        int pierwsza_kostka, druga_kostka;
+        int suma_oczek = 0, suma_pkt = 0;
+        Scanner rzut = new Scanner(System.in);
+        while (numer_rzutu <= 10) {
+            System.out.println("\nGraczu " + g.zwrocImie() + " podaj ilosc oczek w pierwszej kostce (1-6):");
+            pierwsza_kostka = rzut.nextInt();
+            System.out.println("\nGraczu " + g.zwrocImie() + " podaj ilosc oczek w drugiej kostce (1-6):");
+            druga_kostka = rzut.nextInt();
+            /**
+             * Lapanie wyjatkow (blednych danych) i obsluzenie ich.
+             */
+            try {
+                sprawdzPoprawnoscRzutu(pierwsza_kostka);
+            } catch (Exception e) {
+                e.printStackTrace();
+                System.out.println("\nNieprawidlowy rzut graczu " + g.zwrocImie() + " podaj ilosc oczek w pierwszej kostce jeszcze raz(1-6):");
+                pierwsza_kostka = rzut.nextInt();
+            }
+            try {
+                sprawdzPoprawnoscRzutu(druga_kostka);
+
+            } catch (Exception e) {
+                e.printStackTrace();
+                System.out.println("\nNieprawidlowy rzut graczu " + g.zwrocImie() + " podaj ilosc oczek w drugiej kostce jeszcze raz(1-6):");
+                druga_kostka = rzut.nextInt();
+            }
+
+            suma_oczek = pierwsza_kostka + druga_kostka;
+            System.out.println("Gracz " + g.zwrocImie() + " wykonal " + numer_rzutu + " rzut: " + pierwsza_kostka + " " + druga_kostka);
+            System.out.println("Suma oczek na obu kostkach w tym rzucie: " + suma_oczek);
+
+            if (((numer_rzutu == 1) && (suma_oczek == 7)) || (suma_oczek == 11)) {
+                System.out.println("Gracz " + g.zwrocImie() + "wygral ture.");
+                break;
+            } else if (((numer_rzutu == 1) && (suma_oczek == 2)) || (suma_oczek == 12)) {
+                System.out.println("Gracz " + g.zwrocImie() + " przegral ture.");
+                suma_pkt += numer_rzutu;
+                break;
+            } else if (suma_oczek == 5) {
+                System.out.println("Gracz " + g.zwrocImie() + " wygral ture.");
```

Comment:
"Gracz" + player.zwrocImie() -> moznaby to w funkcji trzymac graczPrint(player1, "wygral ture")